xrdp hard coded port for sesman, config settings are not fully read

After changing Globals/ListenPort in sesman.ini to a number other than 3350, clients show an error:
`"Reply from access control: Error connecting to PAM"`
and the next line is logged to xrdp.log:
`[20170926-10:34:23] [ERROR] Failure connecting to socket sesman`

It seems the issue is caused due to the sesman port hard coded to 3350 in xrdp/xrdp_mm.c, access_control(). My colleague  @osalvadorvilardaga has a patch for it which works fine in test environments.

--- a/xrdp/xrdp_mm.c
+++ b/xrdp/xrdp_mm.c
@@ -1471,11 +1471,13 @@ access_control(char *username, char *password, char *srv)
     unsigned long size;
     int index;
     int socket = g_tcp_socket();
+    char port[8];
 
     if (socket != -1)
     {
+        xrdp_mm_get_sesman_port(port, sizeof(port));
         /* we use a blocking socket here */
-        reply = g_tcp_connect(socket, srv, "3350");
+        reply = g_tcp_connect(socket, srv, port);
 
         if (reply == 0)
         {

